packages/react-components/src/organisms/WorkingGroupsTabbedCard.tsx

```diff
@@ -0,0 +1,64 @@
+import { css } from '@emotion/react';
+import React from 'react';
+
+import { WorkingGroupMembership } from '@asap-hub/model';
+import { TabbedCard, WorkingGroupsList } from '../molecules';
+import { rem, tabletScreen } from '../pixels';
+import { splitListBy } from '../utils';
+
+const containerStyles = css({
+  padding: `${rem(32)} 0`,
+  display: 'grid',
+  rowGap: rem(12),
+  [`@media (min-width: ${tabletScreen.min}px)`]: {
+    columnGap: rem(15),
+  },
+});
+
+type WorkingGroupsTabbedCardProps = {
+  userName: string;
+  groups: ReadonlyArray<WorkingGroupMembership>;
+  isUserAlumni: boolean;
+};
+
+const WorkingGroupsTabbedCard: React.FC<WorkingGroupsTabbedCardProps> = ({
+  userName,
+  groups,
+  isUserAlumni,
+}) => {
+  const [inactiveMemberships, activeMemberships] = splitListBy(
+    groups,
+    (group) => isUserAlumni || !group?.active,
+  );
+
+  return (
+    <TabbedCard
```

Comment:
I think the props `getShowMoreText` is required to show the button because we have this  

```
  const displayShowMoreButton =
    getShowMoreText && truncateFrom && items.length > truncateFrom;
```

in `TabbedContent` https://github.com/yldio/asap-hub/blob/master/packages/react-components/src/molecules/TabbedContent.tsx#L55:L56